### PR TITLE
feature/chai-lims-gateway

### PR DESCRIPTION
**Pull request by danielmwakanema**

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { ILIMSConfig } from "src/common/interfaces/lims.config";
+
+export default function (): ILIMSConfig {
+    return {
+        LIMS_API_HOST: process.env.LIMS_API_HOST || 'http://localhost:3000',
```
Comment (bwighane) [inline src/config/index.ts]: Empty values for the defaults usually work well, unless they are meaningful with some values. They may suggest some information about how some implementations look internally i.e when the defaults work in say a staging env or make it a bit difficult to debug in some instances.

File: src/main.ts [hunk 2]
```diff
@@ -1,9 +1,23 @@
 import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 
-async function bootstrap() {
+async function bootstrap(): Promise<void> {
+  const prefix = 'api/v1';
+  const port = process.env.NLIMS_MEDIATOR_PORT || 3000;
   const app = await NestFactory.create(AppModule);
-  app.setGlobalPrefix('nlims/api/v0/fhir');
```
Comment (bwighane) [inline src/main.ts:6]: When creating a channel in the IL, the base path that will separate this mediator is /nlims/api/v0/*.* or something like that. Case in point, it just cannot be /api/v1/ as that would bring about conflicts with the other channels.

**Review by bwighane — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 2 above)
Comment (danielmwakanema) [reply src/main.ts:6]: Okay.

File: src/config/index.ts (hunk 1 above)
Comment (danielmwakanema) [reply src/config/index.ts]: Okay.

**Comment by danielmwakanema** [comment]
I have made the requested changes.

File: src/lims/lims.service.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@nestjs/common';
+import { LIMSClient } from './lims.client';
+import { LabTestResult } from 'src/common/interfaces/lab-test-result';
+
+@Injectable()
+export class LIMSService {
+    constructor(private readonly ohspClient: LIMSClient) {}
+
+    async findPatientById(id: string): Promise<LabTestResult> {
+        return await this.ohspClient.findPatientById(id);
```
Comment (bwighane) [inline src/lims/lims.service.ts:10]: Just a slight variable naming thing, I think it should be reflective. ohspClient -> limsClient

**Review by bwighane — APPROVED** [review]

**Pull request merged**